evm_arithmetization/src/verifier.rs

```diff
@@ -146,25 +146,39 @@ fn verify_proof<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const
         cross_table_lookups,
     } = all_stark;
 
-    let ctl_vars_per_table = get_ctl_vars_from_proofs(
-        &all_proof.multi_proof,
-        cross_table_lookups,
-        &ctl_challenges,
-        &num_lookup_columns,
-        all_stark.arithmetic_stark.constraint_degree(),
```

Comment:
This PR also fixes the use of one STARK’s degree for every table (even though it is not an issue in our current settings).